Source/BoundaryConditions/WarpX_PEC.cpp

```diff
@@ -58,18 +58,27 @@ PEC::ApplyPECtoEfield (std::array<std::unique_ptr<amrex::MultiFab>, 3>& Efield,
         // loop over cells and update fields
         amrex::ParallelFor(tex, tey, tez,
             [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+#if (defined WARPX_DIM_RZ)
```

Comment:
This isn't necessary. `k` is already set to zero. This is also the case when running 2D Cartesian.

Reply:
I was getting a warning. Fixed this with amrex::ignore_unused(k) now.